libs/features/mas/web-components/src/variants/variant-layout.js

```diff
@@ -14,6 +21,25 @@ export class VariantLayout {
     }
   }
 
+  updateCardElementMinHeight(el, name) {
```

Comment:
if only used in product.js, pls just put that function there

Reply:
@npeltier, it was supposed to be shared with mini-compare. Done now